{{cookiecutter.repo_name}}/config/settings/common.py

```diff
@@ -99,7 +99,11 @@
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#databases
 DATABASES = {
     # Raises ImproperlyConfigured exception if DATABASE_URL not in os.environ
-    'default': env.db("DATABASE_URL", default="postgres://{% if cookiecutter.windows == 'y' %}localhost{% endif %}/{{cookiecutter.repo_name}}"),
+    {% if cookiecutter.use_docker == 'y' %}
```

Comment:
This won't work if the user has changed username/password with env variables. `entrypoint.sh` already populates all needed env variables, so I see no use case to add this explicitly.